src/components/Focus/Focus.tsx

```diff
@@ -1,3 +1,7 @@
+/*
+`Focus` will automatically focus the first focusable element that it finds in it's children when it is mounted, unless the user specified an `autoFocus` element.
+*/
```

Comment:
This definition, and the one for `TrapFocus` used to live in a readme file that was removed. I'd like to resurface these since they really helped me how `Focus` and `TrapFocus` works.

Reply:
I'm not sure these comments are the best way to document these 🤔 . Do we know if the readme was removed because they were causing issues with the style guide? Wondering if jsdocs style comments like we use for hooks would make more sense.

Reply:
Iirc we never had readmes. When I added these components we didn't want to expose them. Readme would be the place for these comments, if you want to add some documentation you could try adding a `README_INTERNAL.md` file or something along those lines. I'm *sort of* sure we only look for 'README.md'

Reply:
Yup it looks like the style guide is hardcoded to `readme.md`:

https://github.com/Shopify/polaris-styleguide/blob/dbc092cd68b0e4c90836995112403894fabc4e73/scripts/parseComponentTypes.js#L12

Reply:
FYI this is where I found the definitions https://github.com/Shopify/polaris-react/commit/633c4595880c21216d4cacf2323451b68ca2c79a#diff-475a76122fa5410d0a4098dbc54d3e09